pyproject.toml

```diff
@@ -66,30 +65,14 @@ Issues = "https://github.com/fabiocaccamo/django-extra-settings/issues"
 Funding = "https://github.com/sponsors/fabiocaccamo/"
 Twitter = "https://twitter.com/fabiocaccamo"
 
-[tool.black]
```

Comment:
Please keep the `black` entry and options apart, some editors (eg. Sublime Text / [`python-black`](https://packagecontrol.io/packages/python-black) plugin) do `black` formatting on save based on the presence of `[tool.black]` entry in `pyproject.toml`.